src/NServiceBus.Core/Receiving/ReceiveComponent.cs

```diff
@@ -100,7 +110,9 @@ partial class ReceiveComponent
                     s.ReceiveAddress,
                     TransactionMode = s.RequiredTransportTransactionMode.ToString("G"),
                     s.RuntimeSettings.MaxConcurrency
-                }).ToArray()
+                }).ToArray(),
+                ExternalHandlerRegistry = externalHandlerRegistryUsed,
```

Comment:
All in all I like the changes but I'm on the fence on `ExternalHandlerRegistry`. What is the value of this flag for us or the customer? Is it just the delegate handlers and is that enough to justify having this flag?

Reply:
I figured it was the easiest way to at least leave a paper trail when an external registry is used since that would result in the list of handlers being empty?

Reply:
So the value is to know if no handlers are reported due there being none or due to an external registry being used?